spread.yaml

```diff
@@ -376,6 +385,23 @@ backends:
                   username: external
                   password: ubuntu
 
+    classic-desktop:
+        type: adhoc
+        environment:
+            SPREAD_EXTERNAL_ADDRESS: '$(HOST: echo "${SPREAD_EXTERNAL_ADDRESS:-localhost:8022}")'
+        allocate: |
+            ADDRESS $SPREAD_EXTERNAL_ADDRESS
```

Comment:
What is `ADDRESS` here, is that the spread-provided function?

Reply:
yes it's how the adhoc backend to spread works, you call ADDRESS with what address the machine that was adhoc allocated has